tests/storage-vm

```diff
@@ -609,7 +609,7 @@ for poolDriver in $poolDriverList; do
         lxc config set storage.images_volume "${poolName}"/images
         lxc publish v1 --alias v1image
         lxc delete v1
-        lxc launch v1image v2 -s "${poolName}"
+        lxc launch v1image v2 --vm -s "${poolName}"
```

Comment:
for now we wont want to include this until we decide whether or not we're actually going to make the change, so lets back this out for now.